lib/pf/ConfigStore/EventLogger.pm

```diff
@@ -0,0 +1,50 @@
+package pf::ConfigStore::EventLogger;
+
+=head1 NAME
+
+pf::ConfigStore::EventLogger add documentation
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::ConfigStore::EventLogger
+
+=cut
+
+use HTTP::Status qw(:constants is_error is_success);
+use Moo;
+use namespace::autoclean;
+use pf::file_paths qw($event_loggers_config_file);
+extends 'pf::ConfigStore';
+
+sub configFile { $event_loggers_config_file }
```

Comment:
It doesn't look like there is anything done in the admin interface to manage the event loggers

Reply:
@jrouzierinverse see me if we need expedite a UI section